### PR TITLE
#IS-123 Implement Hello World

### PR DESCRIPTION
**Pull request by brainyfarm**
### What does this PR do?
Implement the hello world feature

### How can this be tested?
.... ....

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log('Hello world')
```
Comment (uaalumni02) [inline app.js:1]: User the logger from the nodejs module rather than console.log

**Review by uaalumni02 — CHANGES_REQUESTED** [review]
Fix the issues highlighted.

File: app.js (hunk 1 above)
Comment (brainyfarm) [reply app.js:1]: Nodejs modules are terrible. Console is better.

**Review by uaalumni02 — APPROVED** [review]
LGTM

**Pull request merged**